exercises/J-functions/exercise.js

```diff
@@ -1,7 +1,25 @@
 function halve(number) {
+  return number/2;
   // complete the function here
 }
 
 var result = halve(12);
+console.log(result);
+
+
```

Comment:
It is unusual to use more than 2 empty lines as a divisor. If you want to show that it's a new section in the code, add a comment.